frontends/web/src/common/Annotation/CreateInterface.js

```diff
@@ -32,6 +32,7 @@ import AnnotationComponent from "./AnnotationComponent.js";
 import initializeData from "./InitializeAnnotationData.js";
 import ResponseInfo from "./ResponseInfo.js";
 import Explainer from "./Explainer.js";
+const yaml = require("js-yaml");
```

Comment:
Do we have to add this package to the package.json ?

Reply:
Oh I probably should. Thanks